openpype/hosts/flame/plugins/publish/integrate_batch_group.py

```diff
@@ -0,0 +1,333 @@
+        include_setup = True
+        # The path attribute where the Open Clip file is exported by
+        # the Write File node.
+        create_clip_path = "<name>"
+        # The path attribute where the Batch setup file
+        # is exported by the Write File node.
+        include_setup_path = "./<name>_v<iteration###>"
+        # The file type for the files written by the Write File node.
+        # Setting this attribute also overwrites format_extension,
+        # bit_depth and compress_mode to match the defaults for
+        # this file type.
+        file_type = "OpenEXR"
+        # The file extension for the files written by the Write File node.
+        # This attribute resets to match file_type whenever file_type
+        # is set. If you require a specific extension, you must
+        # set format_extension after setting file_type.
+        format_extension = "exr"
+        # The bit depth for the files written by the Write File node.
+        # This attribute resets to match file_type whenever file_type is set.
+        bit_depth = "16"
+        # The compressing attribute for the files exported by the Write
+        # File node. Only relevant when file_type in 'OpenEXR', 'Sgi', 'Tiff'
+        compress = True
+        # The compression format attribute for the specific File Types
+        # export by the Write File node. You must set compress_mode
+        # after setting file_type.
+        compress_mode = "DWAB"
+        # The frame index mode attribute of the Write File node.
+        # Value range: `Use Timecode` or `Use Start Frame`
+        frame_index_mode = "Use Start Frame"
+        frame_padding = 6
+        # The versioning mode of the Open Clip exported by the Write File node.
+        # Only available if create_clip = True.
+        version_mode = "Follow Iteration"
+        version_name = "v<version>"
+        version_padding = 3
+
+        # return it as ordered dict
+        reutrn_dict = OrderedDict()
+        # need to make sure the order of keys is correct
+        for item in (
+            ("name", name),
+            ("media_path", media_path),
+            ("media_path_pattern", media_path_pattern),
+            ("create_clip", create_clip),
+            ("include_setup", include_setup),
+            ("create_clip_path", create_clip_path),
+            ("include_setup_path", include_setup_path),
+            ("file_type", file_type),
+            ("format_extension", format_extension),
+            ("bit_depth", bit_depth),
+            ("compress", compress),
+            ("compress_mode", compress_mode),
+            ("frame_index_mode", frame_index_mode),
+            ("frame_padding", frame_padding),
+            ("version_mode", version_mode),
+            ("version_name", version_name),
+            ("version_padding", version_padding)
+        ):
+            reutrn_dict.update({item[0]: item[1]})
+
+        return reutrn_dict
```

Comment:
You can just pass the tuples into initialization of `OrderedDict`.
```suggestion
        # return it as ordered dict
        return OrderedDict(
            ("name", name),
            ("media_path", media_path),
            ("media_path_pattern", media_path_pattern),
            ("create_clip", create_clip),
            ("include_setup", include_setup),
            ("create_clip_path", create_clip_path),
            ("include_setup_path", include_setup_path),
            ("file_type", file_type),
            ("format_extension", format_extension),
            ("bit_depth", bit_depth),
            ("compress", compress),
            ("compress_mode", compress_mode),
            ("frame_index_mode", frame_index_mode),
            ("frame_padding", frame_padding),
            ("version_mode", version_mode),
            ("version_name", version_name),
            ("version_padding", version_padding)
        )
```

Reply:
Didn't find this  the documentation. thanks  for suggestion ;)